Code/Framework/AzCore/AzCore/Asset/AssetManager.cpp

```diff
@@ -2163,24 +2205,24 @@ namespace AZ::Data
         if (curIter != m_assetContainers.end())
         {
             auto newRef = curIter->second.lock();
-            if (newRef && newRef->IsValid())
+            if (newRef && newRef->IsValid() && !isReload) // Reloads already handle when to allow duplicate requests.  Avoid de-duplication here
```

Comment:
nit: Not sure I understand the this comment. Not sure why, we don't use the existing assetContainer here, is it because we want to have a brand new one on a reload event? 

"Avoid de-dupliation here" part I find confusing, when we are duplicating assetContainers on a reload.

Reply:
So the reload logic is already handling cases where it does and does not allow additional requests.  For example, if it sees a load already started, it will allow an additional request to be queued because its possible that the data which is being loaded currently has been updated and is now old, so we need a new request to start over again with the new data.  In that case, we'd end up with 2 containers for the same AssetId.  If it sees the load is just queued but hasn't started loading, it won't even request a container, it will just exit out.  That's why this logic can't be applied in the case of a reload.  I'll see if I can expand on the comment to be more clear.